src/presentation/components/team/Member.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="team">
```

Comment:
As I remember we use css modules

Reply:
➕

Reply:
see how classes are used [here](https://github.com/codex-team/notes.web/blob/main/src/presentation/pages/Settings.vue#L8)